src/Components/LocationModal.tsx

```diff
@@ -1,34 +1,170 @@
 import React from 'react';
 import {
+  Avatar,
+  Button,
   Card,
-  CardContent,
-  Typography,
   CardActions,
-  Button,
-  Modal,
+  CardContent,
   CardHeader,
-  Link,
+  Chip,
+  Collapse,
+  createStyles,
+  Divider,
+  Grid,
+  IconButton,
   List,
   ListItem,
   ListItemAvatar,
-  Avatar,
   ListItemText,
-  IconButton,
-  Divider,
+  Modal,
+  Theme,
+  Typography,
 } from '@material-ui/core';
-
+import clsx from 'clsx';
 import InfoIcon from '@material-ui/icons/Info';
-import LanguageIcon from '@material-ui/icons/Language';
-import CloseIcon from '@material-ui/icons/Close';
+import PhoneIcon from '@material-ui/icons/Phone';
+import DirectionsIcon from '@material-ui/icons/Directions';
+import ShareIcon from '@material-ui/icons/Share';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
 import ReactGA from 'react-ga';
+import { indigo, orange, red } from '@material-ui/core/colors';
+import { makeStyles } from '@material-ui/core/styles';
+import Link from '@material-ui/core/Link';
+import {
+  faCampground,
+  faCircle,
+  faClinicMedical,
+  faFirstAid,
+  faHospital,
+  faMedkit,
+  faShieldAlt,
+  faStethoscope,
+  faUserMd,
+  faVial,
+  faTasks,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
 import { labelMap } from '../App';
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
```

Comment:
This is a big enough style block that I feel it should be in its own file, just to reduce cognitive overhead. We could also use styled-components here, we should settle on a standard we'll be using for styling stuff.